components/tikv_util/src/lib.rs

```diff
@@ -477,6 +478,10 @@ pub fn set_panic_hook(panic_abort: bool, data_dir: &str) {
                 .location()
                 .map(|l| format!("{}:{}", l.file(), l.line()));
             let bt = backtrace::Backtrace::new();
+
+            // redirect to stderr file
+            logger::redirect_panic_log(&panic_log_file);
```

Comment:
I have a question for this: error logs can still occur in `tikv.log` or not? We prefer `dup` instead of `redirect`. So I think the correct way is:
```rust
let panic_logger = create_another_logger();
crit!(panic_logger, msg, ...)
```

Reply:
`create_another_logger` always right, I will change to creat a new logger